### PR TITLE
New package: babashka-0.0.88-2

### PR DESCRIPTION
**Pull request by patrickpichler**

### CODE REVIEW

File: srcpkgs/babashka/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+version=0.0.88
+revision=2
+archs="x86_64"
+build_style=fetch
+hostmakedepends="unzip"
+short_desc="Native fast-starting Clojure scripting environment"
+maintainer="Patrick Pichler <mail@patrickpichler.dev>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/borkdude/babashka"
+distfiles="https://github.com/borkdude/babashka/releases/download/v${version}-${revision}/babashka-${version}-${revision}-linux-static-amd64.zip"
```
Comment (daniel-eys) [inline srcpkgs/babashka/template:12]: Please try to build from source. This is just downloading a precompiled binary.

File: srcpkgs/babashka/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'babashka'
+pkgname=babashka
+version=0.0.88
+revision=2
```
Comment (daniel-eys) [inline srcpkgs/babashka/template:4]: This refers to the template's revision not upstream's, so this should start at 1.

File: srcpkgs/babashka/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+archs="x86_64"
+build_style=fetch
+hostmakedepends="unzip"
+short_desc="Native fast-starting Clojure scripting environment"
+maintainer="Patrick Pichler <mail@patrickpichler.dev>"
+license="GPL-3.0-or-later"
```
Comment (daniel-eys) [inline srcpkgs/babashka/template:10]: Wrong license, the project states EPL-1.0.

File: srcpkgs/babashka/template (hunk 2 above)
Comment (patrickpichler) [reply srcpkgs/babashka/template:4]: The version is not allowed to have underscores.
How would I model something like this in the template?

File: srcpkgs/babashka/template (hunk 3 above)
Comment (patrickpichler) [reply srcpkgs/babashka/template]: Thank you for pointing out! Should be fixed new

File: srcpkgs/babashka/template (hunk 1 above)
Comment (patrickpichler) [reply srcpkgs/babashka/template:12]: Yes, the build is using GraalVM and I'm not sure how to handle it. 

If this is a no-go we you can close this pull request.

Comment (daniel-eys) [reply srcpkgs/babashka/template:12]: I'm just a contributor to void so eventually, a member has to decide whether shipping these precompiled binaries would be acceptable but the general policy is of course to try to build from source.

Comment (patrickpichler) [reply srcpkgs/babashka/template:12]: Is there an official way on how to get things like this in?

Comment (daniel-eys) [reply srcpkgs/babashka/template:12]: Well, the convention then would be to call this package `babashka-bin` (to indicate that this template is fetching precompiled binaries and not building from source) and convince a void pkg-commiter to merge this. But it's very rare void merges 'not-built-from-source' packages.

Comment (patrickpichler) [reply srcpkgs/babashka/template:12]: Thank you very much for the information

**Pull request closed without merging**